DQMOffline/Trigger/python/BPHMonitor_cfi.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.bphMonitoring_cfi import bphMonitoring
+
+hltBPHmonitoring = bphMonitoring.clone()
+hltBPHmonitoring.FolderName = cms.string('HLT/BPH/Dimuon_10_Jpsi_Barrel/')
+hltBPHmonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.int32 (  200  ),
+  xmin  = cms.double(   -0.5),
+  xmax  = cms.double(200),
+)
+hltBPHmonitoring.histoPSet.phiPSet = cms.PSet(
+  nbins = cms.int32 (  64  ),
+  xmin  = cms.double(   -3.2),
+  xmax  = cms.double(3.2),
+)
+hltBPHmonitoring.histoPSet.etaPSet = cms.PSet(
```

Comment:
is this binning enough ?
shouldn't you have a binning more related to the detector features as well ?